app/config/variables.php

```diff
@@ -305,6 +305,46 @@
                 'required' => false,
                 'question' => '',
             ],
+            [
+                'name' => '_APP_STORAGE_DEVICE',
+                'description' => 'Select default storage device',
```

Comment:
Please add more info here, this goes directly to our docs. Also list the different options and mention what is the default value.